go-controller/pkg/config/config.go

```diff
@@ -1327,6 +1332,12 @@ var MetricsFlags = []cli.Flag{
 		Usage:       "Enables metrics related to scaling",
 		Destination: &cliConfig.Metrics.EnableScaleMetrics,
 	},
+	&cli.DurationFlag{
```

Comment:
we dont normally expose this - whats the use case?

Reply:
we may configure this value lesser than 10s, it may be helpful in detecting ipsec tunnel flaps if that happens within 10 secs, could be useful for troubleshooting d/s CI disruptive events if that is caused by ipsec tunnel establishment failures.